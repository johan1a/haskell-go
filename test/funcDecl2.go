package main;

type yeah string
func main(){
}
