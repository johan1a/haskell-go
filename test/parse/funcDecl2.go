package main
import "fmt";
func main( x int  ) 
