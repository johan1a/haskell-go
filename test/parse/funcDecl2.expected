SourceFile (Package "main") [ImportDecl [I1 "fmt"]] [TopLevelDecl2 (FunctionDecl1 "main" (Signature1 [ParameterDecl2 [IdDecl "x"] (TypeName (TypeNameIdentifier "int"))]))]
